src/components/utils/index.js

```diff
@@ -26,6 +26,15 @@ export const preventDefaultOnClick = callback => event => {
 // https://stackoverflow.com/questions/58019463/how-to-detect-device-name-in-safari-on-ios-13-while-it-doesnt-show-the-correct
 const isIOS = (/iPad|iPhone|iPod/.test(navigator.platform || '') || (navigator.platform === 'MacIntel' && navigator.maxTouchPoints > 1)) && !window.MSStream
 
+// WARN: only intended to target Safari 13.1 which has a reported bug handling enumerateDevices()
+// https://bugs.webkit.org/show_bug.cgi?id=209580
+export const isSafari131 = () => {
```

Comment:
[Minor] You execute this function multiple times during the app run. However, I believe the outcome would always be the same. I believe that better is to execute it once during the booting of an app, then store it as a variable and use it within components.

Reply:
Good point. I was having a think about it last night & had the same thought.